src/VisualStudio/Core/Def/Implementation/PreviewPane/PreviewPane.xaml

```diff
@@ -61,6 +62,36 @@
                 </Setter.Value>
             </Setter>
         </Style>
+        <Style x:Key="PreviewPaneRunStyle" TargetType="Run">
+            <Setter Property="Foreground" Value="{DynamicResource {x:Static vsui:EnvironmentColors.CommandBarTextActiveBrushKey}}" />
+            <Style.Triggers>
+                <DataTrigger Binding="{Binding Path=ParentElement.IsKeyboardFocusWithin, RelativeSource={RelativeSource Mode=FindAncestor, AncestorType={x:Type pp:PreviewPane}}}" Value="True">
+                    <Setter Property="Foreground" Value="{DynamicResource {x:Static vsui:EnvironmentColors.CommandBarTextSelectedBrushKey}}" />
+                </DataTrigger>
+            </Style.Triggers>
+        </Style>
+        <Style x:Key="PreviewPaneHyperlinkStyle" TargetType="Hyperlink">
+            <Setter Property="Foreground" Value="{DynamicResource {x:Static vsui:EnvironmentColors.ControlLinkTextBrushKey}}" />
+            <Setter Property="TextDecorations" Value="{x:Null}" />
+            <Setter Property="FocusVisualStyle" Value="{x:Null}" />
+            <Style.Triggers>
+                <Trigger Property="IsMouseOver" Value="True">
+                    <Setter Property="Foreground" Value="{DynamicResource {x:Static vsui:EnvironmentColors.ControlLinkTextHoverBrushKey}}" />
+                </Trigger>
+                <Trigger Property="IsEnabled" Value="False">
+                    <Setter Property="Foreground" Value="{DynamicResource {x:Static vsui:EnvironmentColors.PanelHyperlinkDisabledBrushKey}}" />
+                </Trigger>
+                <Trigger Property="IsEnabled" Value="True">
+                    <Setter Property="Cursor" Value="Hand" />
+                </Trigger>
+                <Trigger Property="IsKeyboardFocused" Value="True">
+                    <Setter Property="TextDecorations" Value="Underline" />
+                </Trigger>
+                <DataTrigger Binding="{Binding Path=ParentElement.IsKeyboardFocusWithin, RelativeSource={RelativeSource Mode=FindAncestor, AncestorType={x:Type pp:PreviewPane}}}" Value="True">
+                    <Setter Property="Foreground" Value="{DynamicResource {x:Static vsui:EnvironmentColors.CommandBarTextSelectedBrushKey}}" />
```

Comment:
Full disclaimer, in the light theme for example, this makes it so these hyperlinks turn black when focused (you have to tab past Preview Changes | Document | Project | Solution to get the focus back up to the top). This is exactly what we want for the High Contrast themes, but sub-optimal for the regular themes. Given the timeline (doing this the right way would require a new ShellColor in the VS repo to be coordinated with this change) and accessibility correctness of this approach, I vote we accept this very minor drawback.

Reply:
Why does it need a new color, there are no suitable existing colors?

Reply:
sounds minor indeed, can you please show a screenshot?

Reply:
Sure.

![lightbulblinkcolors](https://user-images.githubusercontent.com/235241/35817238-b12d1666-0a51-11e8-9b68-1d396eca951c.gif)

Reply:
Looks ok to me, although I don't understand why you cannot use the same color as "Preview Changes", "Document" etc links in the bottom panel.

Reply:
it's a little ugly :(